api-docs/access-token.md

```diff
@@ -20,17 +20,24 @@ Visit the Manager application to create an API User and obtain OAuth 2.0 credent
 > 
 > API Users will act as the admin user. Only the admin user can access the "API Access"-page to create API Users and obtain credentials.
 
-Find the "API Access" page in the upper right corner of the Manager application.
+Find the "API Access" page under Settings → Administration.
 
-![](https://files.readme.io/af2792b-d4792b9-a8de0f7-Skrmbillede_2020-04-15_kl._12.33.13.png "d4792b9-a8de0f7-Skrmbillede_2020-04-15_kl._12.33.13.png")
+![Screenshot of snippet showing API Access under Administration Settings](https://cdn.statically.io/gh/trackunit/developer-hub/master/api-docs/api-access-admin-settings.png "API Access settings")
```

Comment:
Should these links work already now? They don't work when I copy paste them into a browser 🤔

Reply:
No, they'll work when merged to master